packages/fast-animation/.npmignore

```diff
@@ -1,5 +1,10 @@
-client
-webpack
+# Source files
+client/
```

Comment:
could we add __test__/, node_modules, coverage, tslint.json in here as well
can remove .eslintrc, we dont appear to have that anymore

Reply:
I'll add test/, coverage/ and tslint...node modules is never touched in npm publish, even if you try and forcefully include it.